lucid/misc/io/reading.py

```diff
@@ -199,11 +199,11 @@ def _read_and_cache(url, mode="rb"):
 
 
 from functools import partial
-from io import DEFAULT_BUFFER_SIZE
+_READ_BUFFER_SIZE = 1048576
 
 
 def _file_chunk_iterator(file_handle):
-    reader = partial(file_handle.read, DEFAULT_BUFFER_SIZE)
+    reader = partial(file_handle.read, _READ_BUFFER_SIZE)
     file_iterator = iter(reader, bytes())
     # TODO: once dropping Python <3.3 compat, update to `yield from ...`
```

Comment:
Can you in this process also fix the TODO? We're on modern Python these days. Thanks!

Reply:
I don't think we're dropping Python 3.3 explicitly though, so I'd like to defer this for now. I do agree that in practice our team is on modern Python these days.